<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionManagerDialog</class>
 <widget class="QDialog" name="FunctionManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统功能管理</string>
  </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="panelList">
     <layout class="QVBoxLayout" name="verticalLayoutList">
      <item>
       <widget class="QTableWidget" name="tableFunctions"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="panelDetails">
     <layout class="QVBoxLayout" name="verticalLayoutDetails">
      <item>
       <widget class="QGroupBox" name="groupDetails">
        <property name="title">
         <string>功能详情</string>
        </property>
        <layout class="QFormLayout" name="formLayoutDetails">
         <item row="0" column="0">
          <widget class="QLabel" name="labelCurrentName">
           <property name="text">
            <string>名称</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineCurrentName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelLink">
           <property name="text">
            <string>链路</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineLink"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelComponentDependency">
           <property name="text">
            <string>器件依赖</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineComponentDependency"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelAllComponents">
           <property name="text">
            <string>关联器件</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineAllComponents"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="labelFunctionDependency">
           <property name="text">
            <string>功能依赖</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineFunctionDependency"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelExecs">
           <property name="text">
            <string>执行器</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineExecs"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="labelCmdVal">
           <property name="text">
            <string>输入约束</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPlainTextEdit" name="plainCmdVal"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="labelRemark">
           <property name="text">
            <string>备注</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPlainTextEdit" name="plainRemark"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutActions">
        <item>
         <widget class="QPushButton" name="btnAutoDependency">
          <property name="text">
           <string>自动查找依赖功能</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAutoBoundary">
          <property name="text">
           <string>自动查找边界条件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCheckIntegrity">
          <property name="text">
           <string>执行器取反完整性检查</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupOffline">
        <property name="title">
         <string>离线求解结果缓存</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutOffline">
         <item>
          <widget class="QTableWidget" name="tableOffline"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutOfflineButtons">
           <item>
            <widget class="QPushButton" name="btnAddOffline">
             <property name="text">
              <string>新增结果</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemoveOffline">
             <property name="text">
              <string>删除结果</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="offlineSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSaveOffline">
             <property name="text">
              <string>保存离线缓存</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </item>
  <item>
   <layout class="QHBoxLayout" name="horizontalLayoutButtons">
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>新增</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRefresh">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveRecord">
      <property name="text">
       <string>保存更改</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </item>
 </layout>
</widget>
<resources/>
<connections/>
</ui>
